### ISSUE TITLE
-nosteamauth parameter

### ISSUE THREAD

**Issue by TheFreezebug** [Facepunch/garrysmod-requests#908]
Add a parameter for no steam authentication on a server, that or a cVar.  It gets kind of tiring when my VDS is having steam connection issues and I can't connect to my server because it hangs at "Authenticating with steam" 

[http://merp.ga/share/2017-02/2017-02-09_00-23-10.png](http://merp.ga/share/2017-02/2017-02-09_00-23-10.png)

**Comment by uRandomAlex** [comment]
@TheFreezebug Yeah, 2 days ago no one could play multiplayer because the game hanged at "Authenticating with steam".

**Comment by aStonedPenguin** [comment]
This would make spoofing really easy. Pretty sure a2s_* responses are also based on authenticated users as well.
